README.md

```diff
@@ -2,26 +2,21 @@
 
 # Project Name
 
-> Description the project.
-
+> Hello Microverse
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML and CSS
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
 
-
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
```

Comment:
Kindly remove all unused sections inside your README file to make your Readme file look more professional.

![image](https://user-images.githubusercontent.com/68796232/158588756-1fa6f18b-91aa-40c9-af0b-70150a9c9526.png)